File: problemas/iniciante/1065/1065.r
input <- file('stdin', 'r')

pares <- 0
for (i in 1:5){
    numero <- as.integer(readLines(input, n=1))
    if (numero %% 2 == 0){
        pares <- pares + 1
    }
}

write(paste(pares, 'valores pares'), '')

File: problemas/iniciante/1065/1065.go
package main

import (
    "fmt"
)

func main() {
    var pares, numero int

    pares = 0
    for i := 0; i < 5; i++ {
        fmt.Scanf("%d", &numero)

        if(numero % 2 == 0){
            pares += 1
        }
    }

    fmt.Printf("%d valores pares\n", pares)
}
